revocation/result/results.go

```diff
@@ -79,6 +83,71 @@ func NewServerResult(result Result, server string, err error) *ServerResult {
 	}
 }
 
+// CRLReasonCode is CRL reason code (See RFC 5280, section 5.3.1)
+type CRLReasonCode int
+
+const (
+	CRLReasonCodeUnspecified          CRLReasonCode = 0
+	CRLReasonCodeKeyCompromise        CRLReasonCode = 1
+	CRLReasonCodeCACompromise         CRLReasonCode = 2
+	CRLReasonCodeAffiliationChanged   CRLReasonCode = 3
+	CRLReasonCodeSuperseded           CRLReasonCode = 4
+	CRLReasonCodeCessationOfOperation CRLReasonCode = 5
+	CRLReasonCodeCertificateHold      CRLReasonCode = 6
+	// value 7 is not used
```

Comment:
nit: since the values are explicitly set, it is not necessarily to mention "value 7 is not used" as details can be found in the RFC doc.